app/client/src/widgets/TableWidgetV2/widget/__tests__/derived.test/htmlColumns.test.js

```
@@ -0,0 +1,579 @@
+      {
+        id: 2,
+        name: "Usain Bolt",
+        status: "<span style='color: yellow;'>Pending</span>",
+        __originalIndex__: 1,
+      },
+    ];
+
+    let result = getFilteredTableData(input, moment, _);
+
+    expect(result).toStrictEqual(expected);
+    delete input.searchText;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)